### PR TITLE
Add Fleet issue template

### PR DESCRIPTION
**Pull request by olblak**
I propose to add issue templates to encourage contributors to provide information that are useful to debug.

Feel free to look on this [epinio/docs](https://github.com/epinio/docs/issues/new/choose) to see what it would look like

Signed-off-by: Olivier Vernin <olivier.vernin@suse.com>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1-bug-report.yml
Comment (mattfarina) [inline .github/ISSUE_TEMPLATE/1-bug-report.yml:2]: The current label is `kind/bug` and there is no label for triage. We currently have labels that start with `status/`. Should we have a label like `status/needs-triage`?

File: .github/ISSUE_TEMPLATE/2-feature-request.yml
Comment (mattfarina) [inline .github/ISSUE_TEMPLATE/2-feature-request.yml:2]: Another place as the other triage discussion.

**Review by mattfarina — COMMENTED** [review]
This is a good start. Just a couple things around labels we need to work out.

File: .github/ISSUE_TEMPLATE/1-bug-report.yml
Comment (manno) [reply .github/ISSUE_TEMPLATE/1-bug-report.yml:2]: Is that necessary to have 'kind' and 'status'? Either someone is assigned and left a comment, or not. Are we going to process these labels in an automated way?

Comment (olblak) [reply .github/ISSUE_TEMPLATE/1-bug-report.yml:2]: Automatically assigning labels is useful when trying to filter the type of issues.
I think it's a good idea to reuse existing labels so I am gonna change from "bug" to "kind/status" but I am not sure 

Would it makes sense to replace `triage` by `[zube]: To Triage` ?

File: .github/ISSUE_TEMPLATE/2-feature-request.yml
Comment (olblak) [reply .github/ISSUE_TEMPLATE/2-feature-request.yml]: done

**Comment by olblak** [comment]
Waiting for a final review :)

**Review by manno — APPROVED** [review]

**Pull request merged**
